src/js/integrations/spotlight.ts

```diff
@@ -56,19 +57,26 @@ function sendEnvelopesToSidecar(client: Client, sidecarUrl: string): void {
 
     spotlightEnvelope[1] = envelopeItems as Envelope[1];
 
-    fetch(sidecarUrl, {
-      method: 'POST',
-      body: serializeEnvelope(spotlightEnvelope, makeUtf8TextEncoder()),
-      headers: {
-        'Content-Type': 'application/x-sentry-envelope',
-      },
-      mode: 'cors',
-    }).catch(err => {
-      logger.error(
-        "[Spotlight] Sentry SDK can't connect to Spotlight is it running? See https://spotlightjs.com to download it.",
-        err,
-      );
-    });
+    const xhr = createStealthXhr();
```

Comment:
XMLHttpRequest is always available with Sidecar? because createStealthXhr can throw an exception.

Reply:
`XMLHttpRequest` is always available in RN as far as I know.

Reply:
I've changed the throw to null return.